crates/matrix-sdk-base/src/store/memory_store.rs

```diff
@@ -343,25 +343,25 @@ impl MemoryStore {
         }
 
         #[cfg(feature = "experimental-timeline")]
-        for (room, timeline) in &changes.timeline {
+        for (room_id, timeline) in &changes.timeline {
             use tracing::warn;
 
             if timeline.sync {
-                info!("Save new timeline batch from sync response for {room}");
+                info!(%room_id, "Saving new timeline batch from sync response");
             } else {
-                info!("Save new timeline batch from messages response for {room}");
+                info!(%room_id, "Saving new timeline batch from messages response");
             }
 
             let mut delete_timeline = false;
             if timeline.limited {
-                info!("Delete stored timeline for {room} because the sync response was limited");
+                info!(%room_id, "Deleting stored timeline because the sync response was limited");
```

Comment:
You suggested in one place to change this to "Deleted", but suggested "…ing" for most other things. I have applied used the "…ing" form consistently.